src/modules/access-wallet/hybrid/handlers/MEWconnect/index.js

```diff
@@ -29,9 +29,11 @@ class MEWconnectWallet {
     this.mewConnect = await connection.createWalletOnly(
       store.getters['global/network']
     );
-    this.mewConnect.disconnect = () => {
-      this.mewConnect.mewConnect.disconnectRTC();
-    };
+    if (this.mewConnect) {
```

Comment:
I would figure that `await` would prevent this line from happening, is this check for when connection.createWalletOnly is returning null?

Reply:
yep im not 100% sure where the sentry error is coming from - so I just added the check incase but youre right this one shouldn't error out. I'll remove it. I think It is most likely coming from wallet link